src/lib/readdirRecursive.ts

```diff
@@ -51,7 +51,9 @@ export default function readdirRecursive(
 
   const folders = filesInFolder.filter(fileHandle => fileHandle.isDirectory());
 
-  const subFiles = (<string[]>[]).concat(
+  const emptyFilesArray: string[] = [];
```

Comment:
something about the bump from `typescript@5.8.2` to `typescript@5.8.3` caused this to throw due to it not being erasable syntax 🤷🏽